apps/crn-frontend/src/network/working-groups/Outputs.tsx

```diff
@@ -0,0 +1,81 @@
+import { SearchFrame } from '@asap-hub/frontend-utils';
+import { ProfileOutputs } from '@asap-hub/react-components';
+import { useCurrentUserCRN } from '@asap-hub/react-context';
+import { network } from '@asap-hub/routing';
+import { ComponentProps } from 'react';
+
+import { usePagination, usePaginationParams } from '../../hooks';
+import { useWorkingGroupById } from './state';
+
+type OutputsListProps = Pick<
+  ComponentProps<typeof ProfileOutputs>,
+  'hasOutputs' | 'userAssociationMember'
+> & {
+  displayName: string;
+  searchQuery: string;
+  filters: Set<string>;
+  workingGroupId: string;
+};
+type OutputsProps = {
+  workingGroupId: string;
+};
+
+const OutputsList: React.FC<OutputsListProps> = ({
+  workingGroupId,
+  hasOutputs,
+  userAssociationMember,
+}) => {
+  const { currentPage, isListView, cardViewParams, listViewParams } =
+    usePaginationParams();
+  const { numberOfPages, renderPageHref } = usePagination(0, 10);
+  return (
+    <ProfileOutputs
+      researchOutputs={[]}
+      numberOfItems={0}
+      numberOfPages={numberOfPages}
+      currentPageIndex={currentPage}
+      renderPageHref={renderPageHref}
+      isListView={isListView}
+      cardViewHref={
+        network({})
+          .workingGroups({})
+          .workingGroup({ workingGroupId })
+          .outputs({}).$ + cardViewParams
+      }
+      listViewHref={
+        network({})
+          .workingGroups({})
+          .workingGroup({ workingGroupId })
+          .outputs({}).$ + listViewParams
+      }
+      hasOutputs={hasOutputs}
+      userAssociationMember={userAssociationMember}
+      publishingEntity={'Working Group'}
+    />
+  );
+};
+
+const Outputs: React.FC<OutputsProps> = ({ workingGroupId }) => {
+  const hasOutputs = false;
+  const workingGroup = useWorkingGroupById(workingGroupId);
+
+  const currentUserId = useCurrentUserCRN()?.id;
+  const userAssociationMember = !!useWorkingGroupById(
+    workingGroupId,
+  )?.members.filter((member) => member.user.id === currentUserId).length;
+  return (
+    <article>
+      <SearchFrame title="">
+        <OutputsList
+          workingGroupId={workingGroupId}
+          searchQuery={''}
```

Comment:
Why is this stubbed? Won't this mean that the search bar wont allow you to type?

Reply:
Search bar is removed completely until we deal with the ticket in which we actually want to display the research output list. As we are only displaying the NoResearchOutputs page right now, I thought it would be better to remove the search bar component and to pass in a empty value.